### ISSUE TITLE
Add refund support

### ISSUE THREAD

**Issue by dechov** [Automattic/woocommerce-payments#13]
Declare support and implement initial mechanism for full and partial "automatic refunds" [[#](https://docs.woocommerce.com/document/woocommerce-refunds/#section-1)] from the Order screen.

(We currently plan have alternative UI for refunds in the Payments dashboard as well, but this can be scoped as a separate issue once initial support is ready.)
